patches/minecraft/net/minecraft/client/renderer/BlockFluidRenderer.java.patch

```diff
@@ -0,0 +1,30 @@
+--- ../src-base/minecraft/net/minecraft/client/renderer/BlockFluidRenderer.java
++++ ../src-work/minecraft/net/minecraft/client/renderer/BlockFluidRenderer.java
+@@ -1,14 +1,13 @@
+ package net.minecraft.client.renderer;
+ 
+-import net.minecraft.block.Block;
+ import net.minecraft.block.BlockLiquid;
+ import net.minecraft.block.material.Material;
++import net.minecraft.block.state.BlockFaceShape;
```

Comment:
Patches to Minecraft code should not change the imports.
Instead of importing, use the fully qualifies name where you need it, like `net.minecraft.block.state.BlockFaceShape.SOLID`